test/integration/stock/stock.js

```diff
@@ -67,39 +67,39 @@ describe('(/stock/) The Stock HTTP API', () => {
   // list all stock exit relatives to 'Depot Principal'
   it(
     `GET /stock/lots/movements?is_exit=1&depot_uuid=...
-    returns exits for Depot Principal (5 OUT)`,
+    returns list of lot exits for Depot Principal`,
     async () => {
       const res = await agent.get(`/stock/lots/movements?is_exit=1&depot_uuid=${shared.depotPrincipalUuid}`);
```

Comment:
Just as a quick note (no fix needed here) - an easier way to write these queries is to break them into the base url + a query object in `chai`.  It looks like this:

```js
const res  = await (agent.get('/stock/lots/movements')
  .query({ is_exit : 1, depot_uuid : shared.depotPrincipalUuid }));
```

It just makes parameters easier to edit/change and solves the line-length issue.